projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -16,10 +16,11 @@
 		),
 		'allow_jetpack_site_auth' => true,
 		'request_format'          => array(
-			'action'       => '(string) Possible values are \'update\'',
-			'autoupdate'   => '(bool) Whether or not to automatically update the plugin',
-			'active'       => '(bool) Activate or deactivate the plugin',
-			'network_wide' => '(bool) Do action network wide (default value: false)',
+			'action'               => '(string) Possible values are \'update\'',
+			'autoupdate'           => '(bool) Whether or not to automatically update the plugin',
+			'active'               => '(bool) Activate or deactivate the plugin',
+			'network_wide'         => '(bool) Do action network wide (default value: false)',
+			'is_scheduled_updates' => '(bool) If the update is happening as a result of a scheduled update event',
```

Comment:
Let's keep `is_scheduled_update` singular? The update happens in the context of a scheduled update.

Reply:
Sure, thanks for the suggestion!